include/clad/Differentiator/Compatibility.h

```diff
@@ -727,12 +704,17 @@ static inline bool IsPRValue(const Expr* E) { return E->isPRValue(); }
 #define CLAD_COMPAT_CLANG16_CXXDefaultArgExpr_getRewrittenExpr_Param(Node) /**/
 #endif
 
-// Clang 15 rename StringKind::Ascii to StringKind::Ordinary
+// Clang 15 renamed StringKind::Ascii to StringKind::Ordinary
+// Clang 18 renamed clang::StringLiteral::StringKind::Ordinary became clang::StringLiteralKind::Ordinary;
 
 #if CLANG_VERSION_MAJOR < 15
    const auto StringKind_Ordinary = clang::StringLiteral::StringKind::Ascii;
```

Comment:
When there are changes we try to stick to the names in the latest version, for greater proximity to the new versions (easier help search, easier and clearer names, etc.).

```suggestion
   const auto StringLiteralKind_Ordinary = clang::StringLiteral::StringKind::Ascii;
```